python/cudf/cudf/core/dataframe.py

```diff
@@ -3391,7 +3391,7 @@ def quantile(
         numeric_only=True,
         interpolation="linear",
         columns=None,
-        retain_dtype=None,
+        retain_dtypes=None,
```

Comment:
Need to document the default behaviour (i.e., what does `retain_dtype=None` imply?)